polkadot/zombienet_tests/elastic_scaling/0001-basic-3cores-6s-blocks.toml

```diff
@@ -0,0 +1,42 @@
+[settings]
+timeout = 1000
+
+[relaychain.genesis.runtimeGenesis.patch.configuration.config.async_backing_params]
+  max_candidate_depth = 6
+  allowed_ancestry_len = 2
+
+[relaychain.genesis.runtimeGenesis.patch.configuration.config.scheduler_params]
+  max_validators_per_core = 1
+  scheduling_lookahead = 2
+  num_cores = 3
+
+[relaychain.genesis.runtimeGenesis.patch.configuration.config.approval_voting_params]
+  needed_approvals = 3
+  max_approval_coalesce_count = 5
```

Comment:
is this relevant?

Reply:
Not directly relevant to elastic scaling, but it is a good practice for these e2e integration tests to catch more potential issues outside the main focus.